scripts/setup/nrfconnect/get_zap.py

```diff
@@ -118,19 +118,27 @@ def set_executable(location, package_name, filename):
 
 
 def unzip_zap_package(location, package_name):
+    package = location + f"/{package_name}.zip"
+    destination = location + "/" + package_name
+
     try:
-        zip = ZipFile(os.path.join(location, f"{package_name}.zip"))
-        zip.extractall(os.path.join(location, package_name))
-        zip.close()
+        if (platform.system() == 'Darwin'):
+            subprocess.run(['unzip', package, '-d', destination], stdout = subprocess.PIPE)
```

Comment:
Use `subprocess.check_call` or error handling? Note that using `subprocess.PIPE` without consuming the output is dangerous as it may block the child process if PIPE buffer is filled. Btw, what was the issue with `ZipFile`? Should we add a comment for the future reference?

Reply:
Usage of any python library for zip files on the MacOs fails at the unziping Alias files process, i mean it just unzip it as a .txt files. Thi is the reason I change it for the shell command. 
About subprocess.check_call, good point, I will add this.